lib/std/os/bits/darwin.zig

```diff
@@ -1474,3 +1474,25 @@ pub const CLOCK_UPTIME_RAW = 8;
 pub const CLOCK_UPTIME_RAW_APPROX = 9;
 pub const CLOCK_PROCESS_CPUTIME_ID = 12;
 pub const CLOCK_THREAD_CPUTIME_ID = 16;
+
+pub const RUSAGE_SELF = 0;
+pub const RUSAGE_CHILDREN = -1;
+
+pub const rusage = extern struct {
+    utime: timeval,
+    stime: timeval,
+    maxrss: isize,
```

Comment:
are these really signed? What would a negative maxrss mean?

Reply:
They're defined as `long` in [man pages](https://opensource.apple.com/source/man/man-151/usr/man/man2/getrusage.2.auto.html) and [headers](https://github.com/apple/darwin-xnu/blob/master/bsd/sys/resource.h#L187-L201)

These match the existing Linux definitions, which are also longs.